url/resources/a-element-origin.js

```diff
@@ -5,8 +5,7 @@ function setBase(base) {
 }
 
 function bURL(url, base) {
-  base = base || "about:blank"
-  setBase(base)
+  if (base) setBase(base)
```

Comment:
This doesn't work. It will leave the base URL for the document as the URL it is loaded in.

Reply:
Yeah, we would have to skip this type of test when the base is lacking (and say we do it because you cannot unset the base URL of a document).